enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+---
+title: install-ca-management
```

Comment:
This title does not match the goals described blow. The goal is getting the imagestreams working in local registry cases, and this title makes it seem like this is about general ca management during install.

Reply:
it's about managing(consuming) the CAs that the installer collects, which i am referring to as the "install CAs" and hence the management thereof.  Specifically it's about extending what the installer already does with those CAs (distributes to nodes, distributes to proxyconfig) to distribute them to a third location.

So i would say it is indeed "install ca management"

A fully accurate title might be "install CA management for configuring image registry access usage"

I left it somewhat more generic on the assumption that this is likely to be an area of future extension/iteration and that this EP could serve as a more general document for how the install CAs are handled/used/consumed/distributed.

If there already is such an EP, i'd be happy to extend that EP w/ these additions instead.

Reply:
Also it's not only about imagestreams (and components that use them, i.e. web console, samples operator, etc.), but it's also about builds, the integrated registry, and perhaps other components that may access image registries.